lib/ClimaLandSimulations/src/Fluxnet/fluxnet_utilities/site_setup.jl

```diff
@@ -0,0 +1,52 @@
+export site_setup
+
+"""
+    site_setup()
+
+setup site specific domain and time stepping:
+- dz_bottom, the bottom of the soil domain (m)
+- dz_top, the top of the soil domain (m)
+- n_stem, the number of stems
+- n_leaf, the number of leaves
+- h_stem, the height of each stem (m)
+- h_leaf, the height of each leaf (m)
+- t0, the start time of the simulation (s)
+- dt, the time step (s)
+- n, the number of time step between saving outputs
+"""
+function site_setup(;
+    dz_bottom,
+    dz_top,
+    n_stem,
+    n_leaf,
+    h_stem,
+    h_leaf,
+    t0,
+    dt,
+    n,
+)
+    return (
+        dz_bottom = dz_bottom,
+        dz_top = dz_top,
+        n_stem = n_stem,
+        n_leaf = n_leaf,
+        h_stem = h_stem,
+        h_leaf = h_leaf,
+        t0 = t0,
+        dt = dt,
+        n = n,
+    )
+end
+
+function site_setup(site_ID)
+    default_args = Dict(
+        "US-MOz" => ozark_default_args,
+        "US-Ha1" => harvard_default_args,
+        "US-NR1" => niwotridge_default_args,
+        "US-Var" => vairaranch_default_args,
+    )
+
+    if haskey(default_args, site_ID)
```

Comment:
We should add an else case here - maybe raise an error if we don't have default args for the provided `site_ID`, and give a message that this site isn't currently supported (or may have a typo)